tests/artifactoryvirtualrepository_test.go

```diff
@@ -673,3 +675,25 @@ func virtualGenericTest(t *testing.T) {
 	assert.NoError(t, err, "Failed to update "+repoKey)
 	validateRepoConfig(t, repoKey, gvp)
 }
+
+func getVirtualRepoDetailsTest(t *testing.T) {
+	// Create Repo
+	repoKey := generateVirtualRepoKey("go")
+	gvp := services.NewGoVirtualRepositoryParams()
+	gvp.Key = repoKey
+	gvp.RepoLayoutRef = "simple-default"
```

Comment:
If this is not a mandatory fields, let's not set it, because we're not validating it was set anyway.